ajax/libs/vocalizer/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "vocalizer",
+  "filename": "vocalizer.min.js",
+  "version": "1.0.0",
+  "description": "Vocalizer is a plugin that help people learn to say your name",
+  "homepage": "http://www.vocalizer.io",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/atifazam/vocalizer"
+  },
+  "autoupdate": {
```

Comment:
Because there is no any valid tagged released of the repo, git auto-updater will not work. Please give a tag. 
ref: https://git-scm.com/book/en/v2/Git-Basics-Tagging 
Thanks.